src/workspace.ts

```diff
@@ -13,9 +15,9 @@ import {StateManager} from '@cmt/state';
 export class DirectoryContext {
   constructor(
       /**
-       * The directory path that this context corresponds to
+       * Absolute path to the directory associated with this context
```

Comment:
I wouldn't say that this is an "absolute path".  This is a folder object.

Reply:
vector-of-bool's code and comment. Will fix.